dashboard/static/index.html

```
@@ -1675,6 +1683,13 @@ <h2 class="section-page-title">Quality</h2>
         <div class="section-page-header">
           <h2 class="section-page-title">Cost</h2>
         </div>
+        <!-- SPEND CAP. /api/budget reports budget_limit, and it ships NULL:
+             there is no automatic spend stop. Nothing in the UI said so, and
+             the only place that fact surfaced was a bill. Shown ALWAYS, not
+             just when a cap exists, because "no cap" is the state a user most
+             needs to know about. -->
+        <div id="budget-banner" style="display:none;margin-bottom:12px;padding:10px 12px;
```

HTMLHint (1.9.2)

[error] The id value [ budget-banner ] must be unique.

(id-unique)